pyproject.toml

```
@@ -4,7 +4,8 @@ version = "2.0.0"
 description = "Python dependency management and packaging made easy."
 requires-python = ">=3.9,<4.0"
 dependencies = [
-    "poetry-core (==2.0.0)",
+    # TODO: Change to main if poetry-core#805 is merged
+    "poetry-core @ git+https://github.com/finswimmer/core.git@i9961-format",
```

```suggestion
    "poetry-core @ git+https://github.com/python-poetry/poetry-core.git@main",
```

python-poetry/poetry-core#805 is merged.